### DISCUSSION TITLE
Tab Sorting

### DISCUSSION THREAD

**Discussion by Coldblackice** [Chatterino/chatterino2#3596] [Ideas] [answered]
A helpful function would be the ability to right-click on the tab bar and _Sort Alphabetically_ open channels/tabs, as it can become difficult to tell if you already have a channel open amidst the tabs or not. 

An alphabetical sort could be reverted to a _Recently Joined_ sort, or even a _Recently Started (stream)_ sort.

Another possibility is when a user tries to open a new channel/tab, a checkbox that permits Chatterino to take the user to that channel's tab, if already open.

**Comment by jupjohn** [comment]
Not a solution for the sorting problem, but if you're wanting to see if a tab is open you can always use `Ctrl + K` to search for/create it.

**Comment by droserafire** [comment]
I support the alphabetical sorting. I voted for this.

**Reply by 8thony** [reply, marked as answer]
This feature was introduced in #6551 and included in the 2.5.5 release.
